docs/develop/howtos/search_terms_migration.md

```diff
@@ -8,6 +8,10 @@ This is particularly useful when a certain search term is not supported anymore
 
 The query is parsed in the application layer and the old term is replaced with the new one before passing the query to the search engine.
 
+!!! warning "Terms not in the mapping are not considered!"
+    When a search term mapping is defined, other terms (not present in the mapping) are not used by the search engine.
```

Comment:
I would add an example, e.g. if you add a description mapping, then title won't work or something like this.

Reply:
thanks!